tools/setup-envtest/main.go

```diff
@@ -49,10 +49,17 @@ var (
 
 	binDir = flag.String("bin-dir", "",
 		"directory to store binary assets (default: $OS_SPECIFIC_DATA_DIR/envtest-binaries)")
-	remoteBucket = flag.String("remote-bucket", "kubebuilder-tools", "remote GCS bucket to download from")
+
+	useGCS = flag.Bool("use-gcs", false, "use GCS to fetch envtest binaries")
```

Comment:
Not sure if we want to deprecate the GCS functionality. The kubebuilder bucket will definitely go away at some point. We could continue to support downloading from other GCS buckets though